app/frontend/src/components/Navigation/NavButton.tsx

```diff
@@ -8,24 +9,31 @@ interface NavButtonProps {
   route: string;
   label: string;
   labelVariant: Exclude<TypographyProps["variant"], undefined>;
+  notificationCount?: number;
 }
 
 export default function NavButton({
   route,
   label,
   labelVariant,
+  notificationCount,
 }: NavButtonProps) {
   const classes = useNavLinkStyles();
   return (
     <NavLink
       activeClassName={classes.selected}
       exact={route === baseRoute || route === userRoute}
       to={{ pathname: route }}
-      className={classes.link}
+      className={[
+        classes.link,
+        notificationCount ? classes.notification : null,
+      ].join(" ")}
```

Comment:
```suggestion
      className={classNames(classes.link, [classes.notification]: !!notificationCount},
```
And import `classNames`

Reply:
I did not know that classnames was installed, I added it to the last commit. Also I think I get what !!value do but I struggle to understand it's deepest implementation, I mean how it really works.

Reply:
It's the same as `Boolean(value)`, converts to a Boolean. 

If value is truthy (for a number, not zero), !value gives false, so !!value gives true. And the reverse for a falsey value.

Reply:
You don't need to do this conversion in `if (value)`, but in some places, you have to have either true or false, not just truthy or falsey